pkgs/applications/audio/jamulus/default.nix

```diff
@@ -0,0 +1,36 @@
+{ mkDerivation, stdenv, fetchFromGitHub, pkgconfig, qtscript, qmake, libjack2
+}:
+
+mkDerivation {
+  pname = "jamulus";
+  version = "3.5.3";
+  src = fetchFromGitHub {
+    owner = "corrados";
+    repo = "jamulus";
+    rev = "5385f4f113884537c18109e01791149754fe96d5";
+    sha256 = "0wrg0g2cligk3pd82h2sh38xgfbdkwklinrk4p0f4iyfiqfzf0ck";
+  };
+
+  nativeBuildInputs = [ pkgconfig qmake ];
+  buildInputs = [ qtscript libjack2 ];
+
+  buildPhase = ''
+    make clean
+    make
+  '';
```

Comment:
Does the default `buildPhase` not work? Could you add a comment why are you running `make clean`?

Reply:
Thanks for your hints!
the [documentation](https://github.com/corrados/jamulus/wiki/Installation-for-Linux#compile-this-bad-boy) claims

> The "make clean" is essential to remove the automatically generated Qt files which are present in the .tar.gz file and may not match the Qt version you are using.

but actually, it seems to work fine without (with the code from fetchFromGitHub), removed the custom buildPhase now.

Reply:
Unless they commit the files to the repository, `fetchFromGitHub`-fetched source will not contain them so it should be fine without the cleanup.

Reply:
I'm not familiar with Qt development, so no idea what kind of files these are, but I guess it can be ignored here for now, since they don't appear to be present in the specified git rev (since build works & and the binary too...)